docs/src/DatePicker.doc.js

```diff
@@ -0,0 +1,399 @@
+// @flow strict-local
+import * as React from 'react';
+import DatePicker from 'gestalt-datepicker';
+import {
+  arSA,
+  cs,
+  da,
+  de,
+  el,
+  enGB,
+  enUS,
+  es,
+  fi,
+  fr,
+  hi,
+  hu,
+  id,
+  it,
+  ja,
+  ko,
+  ms,
+  nb,
+  nl,
+  pl,
+  ptBR,
+  pt,
+  ro,
+  ru,
+  sk,
+  sv,
+  th,
+  tr,
+  uk,
+  vi,
+  zhCN,
+  zhTW,
+} from 'date-fns/locale';
+import Example from './components/Example.js';
+import PageHeader from './components/PageHeader.js';
+import PropTable from './components/PropTable.js';
+import Combination from './components/Combination.js';
+
+const cards = [];
+const card = c => cards.push(c);
+
+const localeMap = {
+  'ar-SA': { localeData: arSA, lang: ' Arabic (Saudi Arabia)' },
+  'cs-CZ': { localeData: cs, lang: ' Czech' },
+  'da-DK': { localeData: da, lang: ' Danish' },
+  de: { localeData: de, lang: ' German' },
+  'el-GR': { localeData: el, lang: ' Greek' },
+  'en-GB': { localeData: enGB, lang: ' English (British)' },
+  'en-US': { localeData: enUS, lang: ' English (US)' },
+  es: { localeData: es, lang: ' Spanish' },
+  'fi-FI': { localeData: fi, lang: ' Finnish' },
+  fr: { localeData: fr, lang: ' French' },
+  'hi-IN': { localeData: hi, lang: ' Hindi' },
+  'hu-HU': { localeData: hu, lang: ' Hungarian' },
+  'id-ID': { localeData: id, lang: ' Indonesian' },
+  it: { localeData: it, lang: ' Italian' },
+  ja: { localeData: ja, lang: ' Japanese' },
+  'ko-KR': { localeData: ko, lang: ' Korean' },
+  'ms-MY': { localeData: ms, lang: ' Malay' },
+  'nb-NO': { localeData: nb, lang: ' Norwegian (Bokm\u00e5l)' },
+  nl: { localeData: nl, lang: ' Dutch' },
+  'pl-PL': { localeData: pl, lang: ' Polish (Poland)' },
+  'pt-BR': { localeData: ptBR, lang: ' Portuguese (Brazilian)' },
+  'pt-PT': { localeData: pt, lang: ' Portuguese (Portugal)' },
+  'ro-RO': { localeData: ro, lang: ' Romanian' },
+  'ru-RU': { localeData: ru, lang: ' Russian' },
+  'sk-SK': { localeData: sk, lang: ' Slovak' },
+  'sv-SE': { localeData: sv, lang: ' Swedish' },
+  'th-TH': { localeData: th, lang: ' Thai' },
+  tr: { localeData: tr, lang: ' Turkish' },
+  'uk-UA': { localeData: uk, lang: ' Ukrainian' },
+  'vi-VN': { localeData: vi, lang: ' Vietnamese' },
+  'zh-CN': { localeData: zhCN, lang: ' Chinese (Simplified)' },
+  'zh-TW': { localeData: zhTW, lang: ' Chinese (Traditional)' },
+};
+
+card(
+  <PageHeader
+    name="DatePicker"
+    description="Use Datepicker when the user has to select a date or date range."
+    beta
+  />
+);
+
+card(
+  <PropTable
+    props={[
+      {
+        name: 'accessibilityLabelIcon',
+        required: true,
+        type: 'string',
+        description:
+          'Accessibility label for the calendar icon on the date field input that clients such as VoiceOver will read to describe the element.',
+      },
+      {
+        name: 'accessibilityLabelNext',
```

Comment:
@AlbertCarreras can we rename this to `accessibilityLabelNextMonth`?

Reply:
fixed